targets/TARGET_ARM_REV2/src/system_clock_override.c

```diff
@@ -0,0 +1,97 @@
+/* This file overrides the necessary MBED_WEAK functions from system_clock.c to achieve our desired clock config. The
+ * specific version of system_clock.c in the mbed-os repo thats these functions are overriding is based upon the
+ * mbed-xxx target (ex. mbed-stm32f446xe) that this target is linked against in the CMakeLists.txt.
```

Comment:
dis is new. this way we dont need to constantly update the system_clock.c that st is constantly changing. only need to hook the things we actually change